.github/workflows/slim_errors.yml

```diff
@@ -0,0 +1,41 @@
+name: slim_errors
+
+on:
+  pull_request:
+  push:
+    paths-ignore:
+      - '.github/ISSUE_TEMPLATE/**'
+    branches:
+      - master
+
+env:
+  RUSTFLAGS: -Dwarnings --cfg windows_slim_errors
```

Comment:
This sets `windows_slim_errors` for this test run and then tests the `test_result` crate. The generated test.yml will also test `test_result` but without `windows_slim_errors`.